Cargo.toml

```diff
@@ -384,3 +387,7 @@ wrpc-interface-http = { version = "0.29", default-features = false }
 wrpc-runtime-wasmtime = { version = "0.23", default-features = false }
 wrpc-transport = { version = "0.27", default-features = false }
 wrpc-transport-nats = { version = "0.24.1", default-features = false }
+
+[patch.crates-io]
+wasm-pkg-client = { git = "https://github.com/thomastaylor312/wasm-pkg-tools.git", branch = "fix/dependency_override" }
+wasm-pkg-core = { git = "https://github.com/thomastaylor312/wasm-pkg-tools.git", branch = "fix/dependency_override" }
```

Comment:
I think I saw this reflected in the PR, but just make sure we get these removed before merge